Update embedded doc (#25396)
eclipse-ee4j/glassfish: docs/embedded-server-guide/src/main/asciidoc/embedded-server-guide.adoc

        <plugins>
            ...
            <plugin>

>                 <groupId>org.glassfish.embedded</groupId>
>                 <artifactId>maven-embedded-glassfish-plugin</artifactId>

Reviewer: ```suggestion
                <groupId>org.glassfish.embedded</groupId>
                <artifactId>embedded-glassfish-maven-plugin</artifactId>
```

----
version::
  The version to use. The version of the final promoted build for this

>   release is `5.1`. The Maven plug-in is not bound to a specific version

Reviewer: ```suggestion
  release is `7.0`. The Maven plug-in is not bound to a specific version
```

----
<pluginRepositories>
    <pluginRepository>

>         <id>jakarta-oss-staging</id>

Author: removed

http://maven.apache.org/maven-v4_0_0.xsd">

  <modelVersion>4.0.0</modelVersion>
  <groupId>org.example</groupId>
  <artifactId>maven-glassfish-plugin-tester</artifactId>
  <version>1.0.0-SNAPSHOT</version>
  <name>Maven Embedded Glassfish Plugin Example</name>
  <build>
    <plugins>
      <plugin>
        <groupId>org.glassfish.embedded</groupId>

>         <artifactId>maven-embedded-glassfish-plugin</artifactId>

Reviewer: ```suggestion
        <artifactId>embedded-glassfish-maven-plugin</artifactId>
```

>         <version>5.1</version>

Reviewer: ```suggestion
        <version>7.0</version>
```

<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
http://maven.apache.org/maven-v4_0_0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <groupId>org.example</groupId>
    <artifactId>my-ejb-app-tester</artifactId>
    <version>1.0.0-SNAPSHOT</version>
    <name>EJB Embeddable API Example</name>
    <dependencies>
        <dependency>
            <groupId>org.glassfish.main.extras</groupId>

>             <artifactId>glassfish-embedded-static-shell</artifactId>

Reviewer: I would replace rhis dependency with the standard GlassFish Embedded dependency: https://central.sonatype.com/artifact/org.glassfish.main.extras/glassfish-embedded-all/

It’s easier to set up and doesn’t require a local GlassFish installation.

Then also remove GLASSFISH_HOME variable and the system scope definition.

            <version>${project.version}</version>
            <scope>system</scope>
            <systemPath>
                ${env.GLASSFISH_HOME}/lib/embedded/glassfish-embedded-static-shell.jar
            </systemPath>
        </dependency>
        <!--
            The jakarta.jakartaee-api is stripped of any code and is just used to compile your
            application. The scope provided in Maven means that it is used for compiling,
            but is also available when testing. For this reason, the jakartaee-api needs to
            be below the embedded GlassFish dependency. The jakartaee-api can actually be
            omitted when the embedded GlassFish dependency is included, but to keep your
            project Jakarta EE 10 rather than GlassFish specific, specification is important.
        -->
        <dependency>
            <groupId>jakarta.platform</groupId>
            <artifactId>jakarta.jakartaee-api</artifactId>
            <version>10.0.0</version>
            <scope>provided</scope>
        </dependency>
    </dependencies>
    <pluginRepositories>
        <pluginRepository>

>             <id>jakarta-oss-staging</id>

Author: removed

> `dependency` element with the `javaee-api` `artifactId` and the

Author: fixed
